src/index.html

```diff
@@ -11,7 +11,153 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      action="https://NataliaSavenets.github.io/layout_html-form/"
+      method="post"
+    >
+
+      <fieldset class="form__fieldset">
```

Comment:
Don't add empty lines between parent and child, only between siblings

Reply:
Ok. Thank you.